app/controllers/items_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
     @item = Item.new(item_params)
     respond_to do |format|
       if @item.save
-        @item.log('Create Item')
+        current_user.logs << @item.log('Create Item')
```

Comment:
This means, the new created log will assign directly on the current user. (you will see all the created logs will have this)